test/functional/wallet_upgradewallet.py

```diff
@@ -158,40 +170,29 @@ def copy_split_hd():
         self.restart_node(0)
         copy_v16()
         wallet = node_master.get_wallet_rpc(self.default_wallet_name)
-        old_version = wallet.getwalletinfo()["walletversion"]
-
-        # calling upgradewallet without version arguments
-        # should return nothing if successful
-        assert_equal(wallet.upgradewallet(), {})
-        new_version = wallet.getwalletinfo()["walletversion"]
-        # upgraded wallet version should be greater than older one
-        assert_greater_than(new_version, old_version)
+        self.log.info("Test upgradewallet without version arguments")
+        self.test_upgradewallet(wallet, previous_version=159900, requested_version=169900)
```

Comment:
This is not testing what the log claims to test

Reply:
Good catch

Reply:
Fixed